### PR TITLE
Implemented the Channels services for creating, displaying and adding user to a channel and also hooked up the create channel UI with the service

### PR DESCRIPTION
**Pull request by calebjesusegun**

### CODE REVIEW

File: lib/ui/main/channels_creation/channels_creation_view.dart [hunk 1]
```diff
@@ -14,19 +20,27 @@ import 'package:zc_desktop_flutter/ui/shared/dumb_widgets/zcdesk_text.dart';
 class ChannelsCreationView extends HookWidget {
   // const ChannelsCreationView({Key? key}) : super(key: key);
   final GlobalKey<FormState> _formKey = GlobalKey();
+
+    //Declare the services that are dependent upon
+  final _localStorageService = locator<LocalStorageService>();
+
```
Comment (demolaf) [inline lib/ui/main/channels_creation/channels_creation_view.dart:26]: Why isn't this in its own viewmodel

File: lib/ui/main/channels_creation/channels_creation_viewmodel.dart [hunk 2]
```diff
@@ -46,12 +46,14 @@ class ChannelsCreationViewModel extends BaseViewModel with Validator {
   double _iconSize = 22.0;
   bool _isSwitched = false;
```
Comment (demolaf) [inline lib/ui/main/channels_creation/channels_creation_viewmodel.dart:47]: Don't keep view sizes in view model

**Review by demolaf — CHANGES_REQUESTED** [review]
Fix the requested changes and squash your commits

**Review by demolaf — APPROVED** [review]

**Review by osobotu — APPROVED** [review]
LGTM

**Pull request merged**